packages/functional-tests/tests/react-conversion/signInConnectAnotherDevice.spec.ts

```diff
@@ -27,12 +27,6 @@ test.describe('severity-2 #smoke', () => {
       await signinReact.fillOutEmailFirstForm(credentials.email);
       await signinReact.fillOutPasswordForm(credentials.password);
 
-      await signinReact.sendWebChannelMessage(
```

Comment:
Heads up that I filed [FXA-9874](https://mozilla-hub.atlassian.net/browse/FXA-9874) which is related.